src/Summoner/Project.hs

```diff
@@ -153,10 +154,11 @@ generateProject projectName Targets{..} = do
     test   <- decisionToBool isTest "tests"
     bench  <- decisionToBool isBenchmark "benchmarks"
 
-    putStrLn "Latest GHCs: 7.10.3 8.0.2 8.2.2"
-    putStrLn $ "The project will be created with the latest resolver for GHC-" ++ T.unpack defaultGHC
-    testedVersions <- T.words <$>
+    T.putStrLn "Latest GHCs: 7.10.3 8.0.1 8.0.2 8.2.2"
+    T.putStrLn $ "The project will be created with the latest resolver for GHC-" <> showGhcVer defaultGHC
+    testedVersions <- catMaybes . map parseGhcVer . T.words <$>
       queryDef "Additionally you can specify versions of GHC to test with (space-separated): " ""
+    -- TODO: tell which ghc versions were failing to parse?...
```

Comment:
What do you mean?

Reply:
`parseGhcVer` function returns `Maybe GhcVer`. This means that it can fail if user enters something like `8..0.2` or `8.2.3`. Probably some helpful message would be great like:

```haskell
Invalid specified GHC versions: '8..0.2' '8.2.3'
```

And probably if user enters something like `8.2.3` we shouldn't just ignore this input and use default GHC, maybe we should ask for retry...

Reply:
Yes, that's true, something should be done about that..
I guess that just ignoring is not the right thing to do here, as the input could be several versions, and typos are very likely to happen, so the particular error message should be shown and the question re-asked. Are you agree?

Reply:
@vrom911 I think the same. Will to do something about that.